### DISCUSSION TITLE
Donate button in repositories in GitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18951] [Other Feature Feedback, Questions, & Ideas]
Hi all!

#### 1. feature-name
Donate button in repositories in GitHub

#### 2. feature-description 
I wish I had a giveaway button on GitHub

#### 3. Concept
![image](https://user-images.githubusercontent.com/95204402/174529374-786ff5aa-b8aa-4f96-94ba-c8a9f301c60c.png)

#### 4. Why?
1. This allows GitHub to help thousands of developers
2. It's easy to click the donation button is in the visible place

**Comment by Hugoonreplit** [comment]
Hi! I have the Github sponsors and it is very easy to achive. Just visit : 

Github.com/sponsors

There you will find all the things explained.

**Reply by ghost** [reply]
@Hugoondev Hi! How are you? thank you for feeback!

**Reply by Hugoonreplit** [reply]
Hi! I'm good, thank you. How are you?

**Reply by ghost** [reply]
@Hugoondev Hi! I'm good too!
